burn-import/src/onnx/to_burn.rs

```diff
@@ -419,8 +486,54 @@ impl Argument {
     pub fn to_tensor_type(&self) -> TensorType {
         match &self.ty {
             ArgType::Tensor(tensor) => TensorType::new_float(self.name.clone(), tensor.dim),
+            _ => panic!("Can't transform to tensor."),
+        }
+    }
+
+    pub fn to_type(&self) -> Type {
+        match &self.ty {
+            ArgType::Tensor(tensor) => {
+                // Treat tensor with dim 0 as scalar
+                if tensor.dim == 0 {
```

Comment:
Probably not because PyTorch/ONNX treats zero dim as scalar but burn treats tensor with 1 din and shape 1 as scalar. Unless we see a real example of ONNX with tensor.dim == 1 and shape = [1] , we should leave it as as it.